flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/recommendation/ALS.scala

```diff
@@ -25,11 +25,15 @@ import org.apache.flink.api.scala._
 import org.apache.flink.api.common.operators.Order
 import org.apache.flink.core.memory.{DataOutputView, DataInputView}
 import org.apache.flink.ml.common._
-import org.apache.flink.ml.pipeline.{FitOperation, PredictDataSetOperation, Predictor}
+import org.apache.flink.ml.evaluation.RegressionScores
+import org.apache.flink.ml.math.{DenseVector, BLAS}
+import org.apache.flink.ml.pipeline._
 import org.apache.flink.types.Value
 import org.apache.flink.util.Collector
-import org.apache.flink.api.common.functions.{Partitioner => FlinkPartitioner, GroupReduceFunction, CoGroupFunction}
+import org.apache.flink.api.common.functions.{Partitioner => FlinkPartitioner,
+  GroupReduceFunction, CoGroupFunction}
 
+// TODO: Use only one BLAS interface
```

Comment:
I'm not sure if this belongs to this PR. We can get the BLAS operations through ml.math.BLAS, like I did in the predict and evaluate operations, but the lapack ops still need to be done through netlib. Should I change all the BLAS operations to use ml.math.BLAS in this PR?